src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -18,7 +18,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ImportGroup>
 
   <PropertyGroup>
-    <DefaultExcludes>bin\**;obj\**</DefaultExcludes>
+    <DefaultExcludes>bin\**;obj\**;packages\**</DefaultExcludes>
```

Comment:
It does apply to those but the corresponding includes will be filtered by extension (*.cs, *.vb, *.resx). Those won't match xproj. The one exception is that project.json included compiler/resources/**/* (no extension filter) as resources. That's why it's the only case where the xproj exclude matters.